utils/azurescript.sh

```diff
@@ -66,6 +75,10 @@ az cognitiveservices account create --name $COGNITIVE_NAME --resource-group $RG_
 ## Create Database
 az cosmosdb create --name $DB_NAME --kind MongoDB
 
+## Create Storage account for blobs
+az storage account create --name $BLOB_NAME --resource-group $RG_NAME --sku Standard_RAGRS --kind BlobStorage --access-tier Hot
+az storage account blob-service-properties update --enable-delete-retention true --delete-retention-days 100 -n $BLOB_NAME -g $RG_NAME
```

Comment:
Is this going to delete the files after 100 days?

Reply:
This is enabling "soft delete" so files will hang around invisibly if deleted for 100 days
I can put it to a max of 365, but without this setting "soft delete" is OFF by default